### ISSUE TITLE
Add aria-labels on annotations and allow resizing via keyboard

### ISSUE THREAD

**Issue by AbeJellinek** [zotero/zotero#3345]
This should be straightforward for EPUB/snapshot because annotations are rendered via the DOM, but I'm not sure about PDF.
